include/zer_api.h

```diff
@@ -6820,4 +6820,4 @@ typedef struct _zer_callbacks_t
 } // extern "C"
 #endif
 
-#endif // _ZER_API_H
+#endif // _ZER_API_H
```

Comment:
Is this removing the trailing new line?

Reply:
It is yes.

Reply:
Would be nice if the generator script was consistent...